webapi/CopilotChat/Skills/ChatSkills/SemanticChatMemoryExtractor.cs

```diff
@@ -33,11 +34,12 @@ internal static class SemanticChatMemoryExtractor
     /// <param name="kernel">The semantic kernel.</param>
     /// <param name="context">The Semantic Kernel context.</param>
     /// <param name="options">The prompts options.</param>
-    internal static async Task ExtractSemanticChatMemoryAsync(
-        string chatId,
+    /// <param name="logger"></param>
+    [Obsolete]
```

Comment:
Please add the reason this method is obsolete.

Ex: [Obsolete("Use xyz instead...")]